expertsystem/state/particle.py

```diff
@@ -788,23 +789,23 @@ def create_particle(  # pylint: disable=too-many-arguments,too-many-locals
     pid: Optional[int] = None,
     mass: Optional[float] = None,
     width: Optional[float] = None,
-    state: Optional[float] = None,
-    charge: Optional[float] = None,
+    charge: Optional[int] = None,
     spin: Optional[float] = None,
-    isospin: Optional[float] = None,
-    strangeness: Optional[float] = None,
-    charmness: Optional[float] = None,
-    bottomness: Optional[float] = None,
-    topness: Optional[float] = None,
-    baryon_number: Optional[float] = None,
-    electron_lepton_number: Optional[float] = None,
-    muon_lepton_number: Optional[float] = None,
-    tau_lepton_number: Optional[float] = None,
-    parity: Optional[float] = None,
-    c_parity: Optional[float] = None,
-    g_parity: Optional[float] = None,
+    isospin: Optional[Spin] = None,
+    strangeness: Optional[int] = None,
+    charmness: Optional[int] = None,
+    bottomness: Optional[int] = None,
+    topness: Optional[int] = None,
+    baryon_number: Optional[int] = None,
+    electron_lepton_number: Optional[int] = None,
+    muon_lepton_number: Optional[int] = None,
+    tau_lepton_number: Optional[int] = None,
+    parity: Optional[int] = None,
+    c_parity: Optional[int] = None,
+    g_parity: Optional[int] = None,
+    state: Optional[QuantumState[float]] = None,
```

Comment:
@sebastianJaeger Type hints are now checked for the `expertsystem.state.particle` module as well, so I had to fix the type hints here and there.

Reply:
Wah, how did I miss those types in the last PR...